lib/improver/spotdata/common_functions.py

```diff
@@ -35,13 +35,12 @@
 
 import warnings
```

Comment:
Pylint suggests that warnings and iris are not used.

Reply:
I think I have taken these out in another pull request. I'll check.